test/app/model/ChessEngine.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { ChessEngine } from '../../../src/app/model/ChessEngine';
+import { Side, Square, SquareWithPiece } from '../../../src/app/model/Types';
+import { ChessBoard } from '../../../src/app/model/ChessBoard';
+import { Pawn } from '../../../src/app/model/Pawn';
+import { Player } from '../../../src/app/model/Player';
+import 'jest-extended';
+import { Queen } from '../../../src/app/model/Queen';
+
+describe('Chess Engine', () => {
```

Comment:
W testach chcialbym jeszcze zobaczyc, ze to nie dziala tylko dla jednego ruchu :) 
np. zrób move dwa razy i zobacz co będzie wynikiem ostatniego.